controller/Controller.h

```diff
@@ -36,7 +36,7 @@ bool sendSingleCommand(std::unique_ptr<minifi::io::Socket> socket, uint8_t op, c
   minifi::io::BufferStream stream;
   stream.write(&op, 1);
   stream.write(value);
-  return static_cast<size_t>(socket->write(const_cast<uint8_t*>(stream.getBuffer()), gsl::narrow<int>(stream.size()))) == stream.size();
+  return socket->write(const_cast<uint8_t*>(stream.getBuffer()), stream.size()) == stream.size();
```

Comment:
is the `const_cast` needed? as we are touching many of these lines with `const_cast` we could remove them in this PR

Reply:
No, it's not. Will remove them.

Reply:
removed redundant const_casts in 5c6ba7b